### DISCUSSION TITLE
move from container to vanilla z2m: did I mess up my zigbee network???

### DISCUSSION THREAD

**Discussion by bartplessers** [Koenkk/zigbee2mqtt#7138] [General]
Hello,

I was running Z2M as an addon in Home Assistant.
Because of performance issues, I wanted to run it directly on my box
Therefor
- stopped and disabled the Z2M addon
- installed Z2M
- started Z2M with default config.

because I did not get any mqtt messages, I realized that I had to add my network_key and pan_id to the config.
So 
- stopped Z2M
- modified config
- started Z2M

however
- Z2M does not seems to start.  Error message:
`Zigbee2MQTT:error 2021-04-19 21:58:10: Error: Coordinator failed to start, probably the panID is already in use, try a different panID or chann                        el`

So I wanted to revert back to my original config: Z2M in Home Assistant. Tried to start the addon, but:
- Z2M-addon does not seems to start anymore. Error message:
```
Zigbee2MQTT:error 2021-04-19 21:39:23: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-04-19 21:39:23: Failed to start zigbee
Zigbee2MQTT:error 2021-04-19 21:39:23: Exiting...
Zigbee2MQTT:error 2021-04-19 21:39:23: Error: AREQ - ZDO - stateChangeInd after 60000ms
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/dist/utils/waitress.js:46:35)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.18.1 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.18.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

Did I mess up my network? (OH NO, NOT AGAIN....)

So my question is: how can I change from my docker setup to vanilla Z2M, keeping my existing network intact......

any help is appreciated!

kind regards,
B

**Comment by lux73** [comment]
try to shut off all Router Based Devices such as Bulbs, Plug Sockets...   after that Restart z2m - now it should be start...

btw. you had to Copy the /data Folder to the the new dedicated z2m Machine before

**Comment by bartplessers** [comment]
> you had to Copy the /data Folder to the the new dedicated z2m Machine before

Yeah. I noticed that to late.. 🙄

I managed to start my stick again with foiling the whole thing:

![image](https://user-images.githubusercontent.com/10470708/115743020-8a407e80-a391-11eb-8034-a1dbf94bddb8.jpeg)

I(o)T like a pro 🤣
